src/pages/resources/openapi.json

```diff
@@ -7229,7 +7229,10 @@
                 "text",
                 "tables"
               ]
-            }
+            },
+           "default": [
```

Comment:
If we add a default value for elementsToExtract, we should remove it from the required field list.